test/integration/testEasyLoggingOnConnecting.js

```diff
@@ -32,7 +32,7 @@ describe('Easy logging tests', function () {
     resetEasyLoggingModule();
   });
 
-  it('Should apply easy logging config when connection is being opened with callback', function (done) {
+  xit('Should apply easy logging config when connection is being opened with callback', function (done) {
```

Comment:
you can exclude the whole describe function from execution (using `x` or preferrable `describe.skip(...)`)

Reply:
done